inject/inject.vala

```diff
@@ -11,8 +12,9 @@ namespace Frida.Inject {
 	private static bool output_version;
 
 	const OptionEntry[] options = {
-		{ "pid", 'p', 0, OptionArg.INT, ref target_pid, null, "PID" },
-		{ "name", 'n', 0, OptionArg.STRING, ref target_name, null, "PID" },
+		{ "pid", 'p', 0, OptionArg.INT, ref target_pid, "attach to PID", "PID" },
+		{ "name", 'n', 0, OptionArg.STRING, ref target_name, "attach to NAME", "NAME" },
+		{ "file", 'f', 0, OptionArg.STRING, ref spawn_file, "spawn FILE", "FILE" },
```

Comment:
Let's add this one first in the list, for consistency with `frida-tools`.

Reply:
OK